routes/api.php

```diff
@@ -17,3 +18,7 @@
 Route::middleware(['auth:sanctum'])->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::group(['prefix' => '/fenneluckies'], function () {
+    Route::post('/create', [FenneluckyController::class, 'store']);
+});
```

Comment:
The way this is structured right now, it's publicly accessible. Considering adding a middleware — most likely `auth:sanctum` to use the token-based system.

Reply:
Updated. now only authenticated user can create Fenneluckies !